### DISCUSSION TITLE
Provide UI to show recovery instructions

### DISCUSSION THREAD

**Discussion by skrap** [borgbase/vorta#849] [Ideas]
**Is your feature request related to a problem? Please describe.**
When using Vorta, a person might not be aware of exactly the steps to recover from backup.  In particular, the backup key/passphrase and the server location.  (SSH keys as well, but those could often be created anew if lost.)  If a person were to lose their drive which held the backup passphrase, and they did not have it backed up elsewhere, they would lose access to their backups.  This should be 100% clear.

**Describe the solution you'd like**
To make sure a user has all of the information for recovery stored off of the backed-up machine, I suggest that there should be a "recovery instructions" button in the UI, which would produce a document which could be either printed or preserved digitally, and contained:
* A warning that the information in the document is enough to allow anyone to access the backup
* Instructions to store this document securely off of the backed-up machine
* The borg repo location
* The passphrase
* The SSH key (nice to have, see above)
* Other recovery instructions, as appropriate

**Comment by goebbe** [comment]
As a user, I think this proposal would be very helpful. 

Additional use-case: 
At least on Linux based systems, the passphrase of the repos is automatically stored in the "keyring" of the operation system. 
When (re-)installing your Operating-System this passphrase (and the other access data) has to be transferred to the new system - which is far from trivial. 
The suggested "recovery instructions" would provide a very good fallback, to make sure the access to the repo and passphrases are available after reinstallation, even if the transfer of the keyring fails. 

Anyway, there are many use-cases where the repo-passphrase and access-data should be saved at a secure place (outside your computer or even outside your home): 
- hard disk brakes
- laptop is stolen (theft)
- fire incidence in your house - all electronic devices are gone
